helpers/isImageAndTransform.js

```diff
@@ -0,0 +1,38 @@
+import Jimp from "jimp";
+// import fs from "fs/promises";
+// import { storeImage } from "../config/multerConfing.js";
+// import path from "path";
```

Comment:
Jeśli tego nie używasz to proszę to usuń, a jeżeli jest to potrzebne to napisz komentarz nad tym dlaczego.

Reply:
Zapomniałam usunąć, dzięki :)